todo_app/view_model.py

```diff
@@ -0,0 +1,15 @@
+class ViewModel:
+    def __init__(self, items):
+        self._items = items
+    
+    @property
+    def items(self):
+        return self._items
+    
+    @property
+    def todo_items(self):
+        return list(filter(lambda item: item.status == "To Do", self.items))
```

Comment:
Always interesting to see what approach people who've used Python before take - I always find filter a bit ugly and stick to list comprehensions, but it appears I've stumbled upon [[a bit of a minefield](https://stackoverflow.com/questions/3013449/list-comprehension-vs-lambda-filter) looking for any form of justification!